utils/execTo.js

```diff
@@ -0,0 +1,35 @@
+const {
+  execSync
+} = require(`child_process`);
+
+const execToStr = (cmd) => {
+  try {
+    var _execSync;
+
+    return String((_execSync = execSync(cmd, {
+      encoding: `utf8`
+    })) !== null && _execSync !== void 0 ? _execSync : ``).trim();
+  } catch (error) {
+    console.log(error);
+    return ``;
+  }
+}
+
+const execToInt = (cmd) => {
+  // `parseInt` can return `NaN` for unexpected args
+  // `Number` can return undefined for unexpected args
+  // `0 | x` (bitwise or) will always return 0 for unexpected args, or 32bit int
```

Comment:
It looks like we're no longer relying on a bitwise or, so these comments are no longer accurate.

Reply:
yes i removed that as part of the last review so i'll remove the comments 🥳